CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# v3.2.0 (2024-11-06)
+
+## Orchestrator Changes
+### Kubernetes
+* Match the EKS Optimized AMIs secure TLS ciphers ([#230], [#245])
+
+## OS Changes
```

Comment:
This is missing 247:

https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/247